tests/phpunit/tests/admin/includesPlugin.php

```diff
@@ -33,6 +33,36 @@ public function test_get_plugin_data() {
 			$this->assertArrayHasKey( $name, $data );
 			$this->assertSame( $value, $data[ $name ] );
 		}
+
+		$data = get_plugin_data( DIR_TESTDATA . '/plugins/requirescp.php' );
+
+		$default_headers = array(
+			'Name'        => 'With requires CP and without Update URI',
+			'RequiresCP'  => '1.5',
+			'UpdateURI'   => 'https://directory.classicpress.net/wp-json/wp/v2/plugins?byslug=plugins',
```

Comment:
If the core code uses the original plugin slug should this link not reflect that same and be:
`https://directory.classicpress.net/wp-json/wp/v2/plugins?byslug=requirescp`

Or am I misunderstanding the code?

And should this show on the endpoint as a test url - for future proofing against a plugin having the same name?